### ISSUE TITLE
Paperwork-shell. Import empty directory

### ISSUE THREAD

**Issue by ovv** [openpaperwork/paperwork#582]
#214

> Hello
While using the import command to import an empty directory paperwork ask for an importer to use but the list is empty so everything throw an error.
I think printing that the directory is empty and exiting paperwork would be better.
Thanks for the great job you are doing
My use case is to run a cron on a directory where I could dump file to be imported. And is there a way to run OCR with paperwork-shell or is the GUI needed for that ?

> Not raising an exception on empty directories makes sense.
And the OCR should be run automatically when the files are imported (if PDF, only if they don't already contain text). If you want to run explicitly the OCR on a document (for instance PDF with crappy text), it will require a new command in paperwork-shell.
But please open new tickets for those feature (import of empty directory + ocr). This ticket was for Paperwork 1.1 and is closed.

I can provide a pull request for this if you like.

**Comment by jflesch** [comment]
Feel free to send a pull request :-)
(please just beware of the PEP8))

**Comment by jflesch** [comment]
https://github.com/openpaperwork/paperwork-backend/pull/3

**Comment by jflesch** [comment]
Why did you close this ticket ? oO

**Comment by jflesch** [comment]
Fixed at the same time than #593:
https://github.com/openpaperwork/paperwork-backend/commit/35e45fd5eb0897b123ac23c29f43b7923c99afde#diff-c923b83553379c7d9ac0592ce69ad176R396

Will return an error like this:
```json
$ paperwork-shell -bq import ~/tmp/empty
{
    "args": "(\"Don't know how to import ['file:///home/jflesch/tmp/empty']\",)",
    "exception": "<class 'Exception'>",
    "reason": "Don't know how to import ['file:///home/jflesch/tmp/empty']",
    "status": "error"
}
```
